front/lib/api/assistant/visualization.ts

```diff
@@ -20,104 +19,65 @@ export async function getVisualizationPrompt({
   auth: Authenticator;
   conversation: ConversationType;
 }) {
-  const readFirstFiveLines = (inputStream: Readable): Promise<string[]> => {
-    return new Promise((resolve, reject) => {
-      const rl: readline.Interface = readline.createInterface({
-        input: inputStream,
-        crlfDelay: Infinity,
-      });
+  const isJITEnabled = !(await isJITActionsEnabled(auth));
```

Comment:
I don't understand the condition. if we `!` the function, then I don't understand the function name

Reply:
it's a mistake from me updating the name late. Thanks for catching.